tests/jsTests/validationSpec.js

```diff
@@ -0,0 +1,19 @@
+describe('validateNumber', function() {
+    it('Should be phone number', function() {
+        expect(validation.isPhoneNumber('342345235')).toBeTruthy();

```

Comment:
Is there any fixed length for the phone number or you just allow any length as long as it contains digits? if you are checking the length, include that as well.